Contents/mods/FuelAPI/media/lua/shared/Translate/PL/ItemName_PL.txt

```diff
@@ -0,0 +1,4 @@
+ItemName_PL = {
+    ItemName_FuelAPI.LargePetrolCan = "Pusty du�y kanister na benzyn�",
```

Comment:
ItemName_FuelAPI.LargePetrolCan = "Pusty, duży kanister na benzynę",

Reply:
special letters are wrong coded in your wersion @Tread4